common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -317,17 +317,22 @@
                 }, error);
             };
 
+            DiscussionThreadListView.prototype.checkImagePost = function(threadBody) {
```

Comment:
It would be good to make sure that this works with MathJax too (it looks like it will). If it does, I'd rename this as something like ``containsMarkup``.